src/features/network-modification-table/network-modifications-table.tsx

```
@@ -91,10 +100,31 @@ export function NetworkModificationsTable({
+        // the new assembled composite will be created where the first selected row is so :
+        // depth has to be < to first selected row depth + maxDepth of any selected row
+        if (rows.length === 0) return false;
+        const firstSelectedRowDepth = findDepth(composedModificationsRef.current, rows[0].uuid);
+        return rows.some((row) => firstSelectedRowDepth + (row.maxDepth ?? 0) >= MAX_COMPOSITE_NESTING_DEPTH);
+    }, []);
+
+    const handleRowSelected = useCallback(
+        (selectedRows: ComposedModificationMetadata[]) => {
+            onSelectedRowsChange(selectedRows, mayModificationsBeAssembled(selectedRows));
+        },
+        [onSelectedRowsChange]
```

ESLint

[error] React Hook useCallback has a missing dependency: 'mayModificationsBeAssembled'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)